man/nvim.1

```diff
@@ -0,0 +1,375 @@
+.Dd May 7, 2015
+.Dt NVIM 1
+.Os Neovim
+.Sh NAME
+.Nm nvim
+.Nd edit text
+.Sh SYNOPSIS
+.Nm
+.Op Ar options
+.Op Ar file ...
+.Nm
+.Op Ar options
+.Fl
+.Nm
+.Op Ar options
+.Fl t Ar tag
+.Nm
+.Op Ar options
+.Fl q Ar errorfile
+.Sh DESCRIPTION
+.Nm
+is a text editor based on
+.Xr vim 1 .
+To enter commands in
+.Nm ,
+type a colon
+.Sy ( \(dq:\(dq ) ,
+which is also used in this manual to denote commands.
+For more information, consult the on-line help system with the
+.Ic :help
+command.
+.Bl -tag -width Fl
+.It Ar file ...
+File(s) to edit.
+If none are specified, open an empty buffer.
+If multiple files are specified, open one buffer for each file.
+To switch between buffers, use the
+.Ic :next
+and
+.Ic :previous
+commands.
+.It Fl
+Read text from standard input until EOF, then open a buffer with that text.
+Commands are read from standard error, which should be a terminal.
```

Comment:
> I think TTY or PTY would be more accurate

I used terminal because it's more generic, and TTY stands for teletype ([among other things apparently](https://en.wikipedia.org/wiki/Teleprinter)), which is probably wrong in most modern contexts.

> but this sounds odd because an output stream isn't a terminal. Maybe "should connect to a terminal".

:+1: perhaps "... , which should be associated to a terminal."?